Makefile

```diff
@@ -41,7 +41,7 @@ samples:
 
 tiny.en tiny base.en base small.en small medium.en medium large-v1 large-v2 large-v3 large-v3-turbo:
 	bash ./models/download-ggml-model.sh $@
-	cmake -B build
+	cmake -B build -DGGML_CUDA=$(GGML_CUDA) -DGGML_MUSA=$(GGML_MUSA)
```

Comment:
Perhaps this could be turned into a single variable like `CMAKE_ARGS`:
```console
cmake -B build $(CMAKE_ARGS)
```
And then instead of adding environment variables to the docker files the cmake options could be specified with the RUN command:
```docker
RUN make base.en CMAKE_ARGS="-DGGML_MUSA=1"
```
This way we can add other options in the future without having to update the Makefile.

Reply:
This is a good idea.

Reply:
I'm just not sure how `ENV GGML_CUDA=1` was working with `RUN make base.en` in `main-cuda.Dockerfile` previously.  
That said, if you'd prefer using a single variable approach, I'm happy to update it accordingly.

Reply:
Please see the latest commit.

Reply:
> I'm just not sure how `ENV GGML_CUDA=1` was working with `RUN make base.en` in `main-cuda.Dockerfile` previously.

I was thinking the same thing. I noticed that it is currently commented out and perhaps it did not work. I'll take a closer look at this soon.